README.md

```diff
@@ -796,6 +796,7 @@ UiAutomator2 driver supports Appium endpoints for applications management:
 - Check if app is installed (`POST /appium/device/app_installed`)
 - Install/upgrade app (`POST /appium/device/install_app`)
 - Active app (`POST /appium/device/activate_app`)
+    - The backend calls `monkey` command. It [turns on auto rotation](https://stackoverflow.com/questions/56684778/adb-shell-monkey-command-changing-device-orientation-lock), so please consider to use [mobile: startActivity](#mobile-startactivity) if you would like to keep the rotation preference.
```

Comment:
we could add a link to the tool docs here: https://developer.android.com/studio/test/other-testing-tools/monkey